src/System.Diagnostics.StackTrace/tests/StackTraceTests.cs

```diff
@@ -321,7 +321,7 @@ public void ToString_Invoke_ReturnsExpected(StackTrace stackTrace, string expect
         public void ToString_NullFrame_ThrowsNullReferenceException()
         {
             var stackTrace = new StackTrace((StackFrame)null);
-            Assert.Throws<NullReferenceException>(() => stackTrace.ToString());
+            Assert.Equal(Environment.NewLine, stackTrace.ToString());
```

Comment:
This test will need to be skipped on netfx, right?

Reply:
Good point!